src/views/AppRoot/__tests__/component.spec.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import AppRootComponent from '../component'
+
+describe('AppRootComponent matches snapshot', () => {
+  const defaultProps = {
+    isLogged: false
+  }
+
+  it('with default props', () => {
```

Comment:
```describe('with default props', () => {```

Reply:
as the summary:
* `describe` is for describing testing entity or testing context, in 2nd case describe should begin with "with", "without", "when" words
* `it` is for expectation, for example: `it('returns correct result), () => ...` or `it('matches snapshot'), () => ...`

Reply:
Good points, thanks